dashboard/app/models/ability.rb

```diff
@@ -249,7 +251,7 @@ def initialize(user)
 
       can [:edit_manifest, :update_manifest, :index, :show, :update, :destroy], :dataset
 
-      can :validate_form, :pd_foorm
+      can [:validate_form, :save_form], :pd_foorm
```

Comment:
I'm not quite sure how this works the way it is written here, but I think it would be more conventional to put the permissions on the methods which are exposed as rails routes, such as `:publish` and `:create`, rather than on the `save_form` helper. if you do this, then `save_form` can become a private method in the controller, to help ensure that it does not get called from outside of the other controller methods.

Reply:
that's a mistake, I had moved the save functionality from one file to another and forgot to update this. Will fix. The Foorm::Form permissions should be what is actually used here.